lib/mpl_toolkits/axes_grid1/anchored_artists.py

```diff
@@ -52,7 +52,11 @@ def __init__(self, width, height, xdescent, ydescent,
             Font property used as a reference for paddings.
 
         frameon : bool, default: True
-            If True, draw a box around this artists.
+            If True, add a frame and solid background for the artist.
+            Frame appearance can be changed with, i.e.
```

Comment:
This is a 'for example' case, not 'that is':
```suggestion
            Frame appearance can be changed with, e.g.,
```